funds-manager/funds-manager-server/src/custody_client/queries.rs

```diff
@@ -27,6 +27,33 @@ impl CustodyClient {
         Ok(())
     }
 
+    /// Find an inactive gas wallet
+    pub async fn find_inactive_gas_wallet(&self) -> Result<GasWallet, FundsManagerError> {
+        let mut conn = self.get_db_conn().await?;
+        gas_wallets::table
+            .filter(gas_wallets::active.eq(false))
```

Comment:
clippy complaining here that there is no member `active` in the `gas_wallets` module - not sure what the diesel macros expand to but `active` doesn't seem to be a column on the `gas_wallets` table anyway, this should be `status` right?

Reply:
yeah this got messed up in the rebase order, it's fixed in one of the follow ups